web/libs/editor/src/utils/canvas.js

```diff
@@ -345,13 +345,27 @@ function Region2RLE(region) {
   return rle;
 }
 
-function brushSizeCircle(size) {
+/**
+ * Creates a custom circular cursor visual representation based on the provided brush size.
+ * Max size: 128px according to browser limitations.
```

Comment:
Does it make sense to add checks for this and limit the `size` in the code?

Reply:
For now we do not have such a big brush size. So I ignored it temporarily.